daemon/src/taker_cfd.rs

```diff
@@ -63,7 +63,7 @@ pub struct Actor {
     send_to_maker: Address<send_to_socket::Actor<wire::TakerToMaker>>,
     monitor_actor: Address<monitor::Actor<Actor>>,
     setup_state: SetupState,
-    latest_announcements: Option<oracle::Announcements>,
+    latest_announcements: Option<BTreeMap<OracleEventId, oracle::Announcement>>,
```

Comment:
Not sure if we need to semantically differentiate between `Option::None` and an empty map, so I think we could get rid of the `Option`.